### DISCUSSION TITLE
24.4.8

### DISCUSSION THREAD

**Discussion by davidjgraph** [jgraph/drawio-desktop#1751] [General]
## Releases Notes for 24.4.8
[Windows Installer](https://github.com/jgraph/drawio-desktop/releases/download/v24.4.8/draw.io-24.4.8-windows-installer.exe)
[Windows No Installer](https://github.com/jgraph/drawio-desktop/releases/download/v24.4.8/draw.io-24.4.8-windows-no-installer.exe)
[macOS - Universal](https://github.com/jgraph/drawio-desktop/releases/download/v24.4.8/draw.io-universal-24.4.8.dmg)
Linux - [deb](https://github.com/jgraph/drawio-desktop/releases/download/v24.4.8/drawio-amd64-24.4.8.deb), [AppImage](https://github.com/jgraph/drawio-desktop/releases/download/v24.4.8/drawio-x86_64-24.4.8.AppImage) or [rpm](https://github.com/jgraph/drawio-desktop/releases/download/v24.4.8/drawio-x86_64-24.4.8.rpm)

Windows intel x32 releases are marked -ia32-

**ChangeLog:**

- #1746 , #1750 
- Uses electron 30.0.6.
- Updates to [draw.io core 24.4.8](https://github.com/jgraph/drawio/blob/v24.4.8/ChangeLog). All changes from 24.4.8 are added in this build.

<hr /><em>This discussion was created from the release <a href='https://github.com/jgraph/drawio-desktop/releases/tag/v24.4.8'>24.4.8</a>.</em>

**Comment by naquashv** [comment]
This is so good. Thanks a ton :]

**Reply by Robintes** [reply]
Thnx for that djgraph solved my print problem

**Comment by mbugueno** [comment]
This software is excellent¡¡ Thanks
